locale/fi/LC_MESSAGES/django.po

```diff
@@ -3625,18 +3626,18 @@ msgstr "{:d} pisteet"
 
 #: userprofile/templates/userprofile/login.html:6
 #: userprofile/templates/userprofile/login.html:17
-msgid "Login"
-msgstr "Kirjaudu sisään"
+msgid "Log in to A+"
```

Comment:
I now realize that this new version of the string hardcodes the BRAND_NAME setting. We will need to change this later since the BRAND_NAME can be customized on each site.